inc/functions-child.php

```diff
@@ -15,4 +15,40 @@ function dequeue_parent_style()
 {
 	wp_dequeue_style('tna-styles');
 	wp_deregister_style('tna-styles');
+}
+
+function get_lessons( $boxes ) {
+	foreach ( $boxes as $box ) { ?>
+		<div class="col-xs-12 col-sm-6">
+			<article>
+				<div class="entry-header">
+					<h2>
+						<a href="<?php echo $box['url']; ?>" title="<?php echo $box['title'] ?>">
+							<?php echo $box['title']; ?>
+						</a>
+					</h2>
+				</div>
+				<div class="entry-content">
+					<?php if ( $box['image'] ) { ?>
+						<a href="<?php echo $box['url']; ?>" class="thumbnail" title="<?php echo $box['title'] ?>">
+							<img src="<?php echo $box['image']; ?>" class="img-responsive" alt="<?php echo $box['title'] ?>">
+						</a>
+					<?php } ?>
+					<p><?php echo $box['excerpt']; ?></p>
+					<?php if ( isset($box['child_pages']) ) { ?>
+						<ul >
```

Comment:
Ln 39 - add class="full" to the UL to use the bulleted navigation style (used for lists of links).

Reply:
Hi Simon, this has been done